Source/WTF/wtf/PlatformUse.h

```diff
@@ -416,8 +416,10 @@
 #define USE_CORE_TEXT_VARIATIONS_CLAMPING_WORKAROUND 1
 #endif
 
-#if PLATFORM(IOS) && !PLATFORM(IOS_FAMILY_SIMULATOR) && __has_include(<ServiceExtensionsCore/SEMemory_Private.h>)
+#if (PLATFORM(IOS) || PLATFORM(MAC)) && !PLATFORM(IOS_FAMILY_SIMULATOR) && __has_include(<BrowserEngineCore/BEMemory.h>)
 #define USE_INLINE_JIT_PERMISSIONS_API 1
+#elif (PLATFORM(IOS) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 170400  && __IPHONE_OS_VERSION_MAX_ALLOWED < 180000)
+#error
```

Comment:
Maybe a word or two here would be nice?

Reply:
Can we just use SDK version:
```c
#if PLATFORM(IOS) && !PLATFORM(IOS_FAMILY_SIMULATOR) && \
__IPHONE_OS_VERSION_MAX_ALLOWED >= 170400 && \
__IPHONE_OS_VERSION_MAX_ALLOWED < 180000
```
as the condition? Testing for the header seems likely to cause this to suddenly flip on other platforms, and the error statement implies to me that we really want to be sure it's enabled in `17.4..<18`